controller/scheduler/swarm.py

```diff
@@ -48,14 +63,20 @@ def stop(self, name):
         """
         Stop a container
         """
-        self.docker_cli.stop(name)
+        try:
+            self.docker_cli.stop(name)
+        except:
+            pass
```

Comment:
don't we want to raise RuntimeError if we fail to stop? I don't think we should be suppressing these errors